mix.exs

```diff
@@ -36,13 +36,13 @@ defmodule Ethereumex.Mixfile do
 
   defp deps do
     [
-      {:httpoison, "~> 1.4.0"},
-      {:jason, "~> 1.1"},
-      {:credo, "~> 0.10.2", only: [:dev, :test], runtime: false},
-      {:ex_doc, "~> 0.19", only: :dev, runtime: false},
-      {:dialyxir, "~> 1.0.0-rc.7", only: [:dev, :test], runtime: false},
+      {:httpoison, "~> 1.6.0"},
+      {:jason, "~> 1.2"},
+      {:credo, "~> 1.3", only: [:dev, :test], runtime: false},
+      {:ex_doc, "~> 0.21", only: :dev, runtime: false},
+      {:dialyxir, "~> 1.0.0", only: [:dev, :test], runtime: false},
       {:poolboy, "~> 1.5.1"},
-      {:telemetry, "~> 0.4", optional: true}
+      {:telemetry, "~> 0.4.1", optional: true}
```

Comment:
Doest it really need to be 0.4.1? Can we use 0.4? The same question for httpoison

Reply:
touche!